src/puppetlabs/puppetdb/scf/storage.clj

```diff
@@ -334,22 +334,30 @@
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;; Catalog updates/changes
 
-(defn catalog-metadata
+(defn latest-catalog-metadata
   "Returns the id and hash of certname's catalog"
```

Comment:
What are you, the comment police? (says me who just made the same note on your PR)